### ISSUE TITLE
how to get the best dice checkpoint during training?

### ISSUE THREAD

**Issue by LiuTingWed** [WuJunde/MedSegDiff#136]
Good job！
But how can i get the best dice checkpoint during training? It seem there is no relative  eval code in the project during training.

**Comment by maomaocun** [comment]
In my work,isic dataset got the best presentation in checkpoint 80000iterations batch_size =8 MedSegdiff-v1 basic,iou=0.8197,dice=0.8981

**Comment by maomaocun** [comment]
yes,just sample from about 60000iterations 65000iteration ...... to 11500iterations,and you can see the 80000iteration did the best ,my gpu is burnning!!!!!!!!!!!!!!

**Comment by maomaocun** [comment]
can I communicate with you in Wechat?
19162076385

**Comment by wzx0720** [comment]
> can I communicate with you in Wechat? 19162076385

@maomaocun Hi, I also do my research on MedSegDiff-v1, but I met some problems while sampling. May I communicate with you on WeChat, too?

**Comment by LiuYM00** [comment]
> 在我的工作中，isic 数据集在 checkpoint 80000 次迭代中得到了最好的表示 batch_size =8 MedSegdiff-v1 basic，iou=0.8197，dice=0.8981

您好，想问一下代码是否进行了修改，我也使用了batch_size=8 medsegdiff-v1,但是效果非常差
